lib/stringifier.d.ts

```diff
@@ -35,7 +35,7 @@ declare class Stringifier_ {
   rawEmptyBody(root: Root): string | undefined
   rawIndent(root: Root): string | undefined
   rawSemicolon(root: Root): boolean | undefined
-  rawValue(node: AnyNode, prop: string): string
+  rawValue(node: AnyNode, prop: string): any
```

Comment:
```suggestion
  rawValue(node: AnyNode, prop: string): unknown
```

`any` is bad practice. If we don’t sure about type we must use `unknown`, so it will force developer to detect type.